pkg/monitortests/testframework/disruptionserializer/write_job_run_data_test.go

```diff
@@ -45,30 +60,75 @@ func TestComputeDisruptionData(t *testing.T) {
 			intervals: []monitorapi.Interval{
 				{
 					Condition: monitorapi.Condition{
-						Level:   monitorapi.Info,
-						Locator: "backend-disruption-name/kube-api-new-connections disruption/kube-api connection/new",
-						Message: "disruption/kube-api connection/new started responding to GET requests over new connections",
+						Level: monitorapi.Info,
+						StructuredLocator: monitorapi.Locator{
+							Type: monitorapi.LocatorTypeDisruption,
+							Keys: map[monitorapi.LocatorKey]string{
+								monitorapi.LocatorBackendDisruptionNameKey: "kube-api-new-connections",
+								monitorapi.LocatorDisruptionKey:            "kube-api",
+								monitorapi.LocatorConnectionKey:            "new",
+							},
+						},
+						StructuredMessage: monitorapi.Message{
+							Reason:       monitorapi.DisruptionEndedEventReason,
+							Cause:        "",
+							HumanMessage: "disruption/kube-api connection/new started responding to GET requests over new connections",
+							Annotations: map[monitorapi.AnnotationKey]string{
+								monitorapi.AnnotationReason: string(monitorapi.DisruptionEndedEventReason),
+							},
+						},
 					},
-					From: time.Now().Add(-60 * time.Minute),
-					To:   time.Now().Add(-30 * time.Minute),
+					From:   time.Now().Add(-60 * time.Minute),
+					To:     time.Now().Add(-30 * time.Minute),
+					Source: monitorapi.SourceDisruption,
 				},
 				{
 					Condition: monitorapi.Condition{
-						Level:   monitorapi.Error,
-						Locator: "backend-disruption-name/kube-api-new-connections disruption/kube-api connection/new",
-						Message: "reason/DisruptionBegan ns/openshift-image-registry route/test-disruption-new disruption/image-registry connection/new stopped responding to GET requests over new connections: Get \"https://test-disruption-new-openshift-image-registry.apps.ci-op-4mb5069g-03fd1.XXXXXXXXXXXXXXXXXXXXXX/healthz\": dial tcp 104.154.53.8:443: connect: connection refused",
+						Level: monitorapi.Error,
+						StructuredLocator: monitorapi.Locator{
+							Type: monitorapi.LocatorTypeDisruption,
+							Keys: map[monitorapi.LocatorKey]string{
+								monitorapi.LocatorBackendDisruptionNameKey: "kube-api-new-connections",
+								monitorapi.LocatorDisruptionKey:            "kube-api",
```

Comment:
We set `LocatorDisruptionKey` to `kube-api` here, but the original `Message` says `disruption/image-registry` -- maybe the original was wrong and should've said `disruption/kube-api`?

lines 177, 225, 310, 382 (with `openshift-api`) also have this inconsistency.